src/support/rand.c

```diff
@@ -131,18 +135,10 @@ __wt_random(WT_RAND_STATE *rnd_state) WT_GCC_FUNC_ATTRIBUTE((visibility("default
 #ifdef ENABLE_ANTITHESIS
     return (uint32_t)(fuzz_get_random());
 #else
-    WT_RAND_STATE rnd;
     uint32_t w, z;
 
-    /*
-     * Generally, every thread should have their own RNG state, but it's not guaranteed. Take a copy
-     * of the random state so we can ensure that the calculation operates on the state consistently
-     * regardless of concurrent calls with the same random state.
-     */
-    rnd = *rnd_state;
-    WT_ACQUIRE_BARRIER();
```

Comment:
The acquire barrier is removed here. Was this intentional?

Reply:
Yes, please see comment above about multithreading.